frontend/packages/knative-plugin/integration-tests/features/eventing/eventing-create-sink-broker.feature

```diff
@@ -0,0 +1,73 @@
+@knative-eventing
+Feature: Knative Eventing Broker Support
+              As a user, I should be able to create broker and sink it with event sources
+
+
+        Background:
+            Given user has installed OpenShift Serverless Operator
+              And user has created Knative Eventing CR
+              And user has created or selected namespace "aut-eventing-broker"
+
+
+        @regression @to-do @odc-5030
+        Scenario: Create Broker form: KC-01-TC01
+            Given user is at create broker form
+             Then user will see Form view default selected
+              And user will see YAML view radio button
+              And user will Application drop down
+              And user will see Name field for broker
+        
+
+        @smoke @to-do @odc-5030
+        Scenario: Create Broker using Form view: KC-01-TC02
+            Given user is at Add page
+             When user selects on Broker from Serverless card
+              And user selects Form view
+              And user enters borker name as "default-broker"
+              And user clicks on Create button
+             Then user will be redirected to Topology page
+              And user will see the "default-broker" broker created
+
+        
+        @regression @to-do @odc-5030
+        Scenario: Create Broker using YAML view: KC-01-TC03
+            Given user is at Add page
+             When user selects on Broker from Serverless card
```

Comment:
```suggestion
             When user selects on Broker from "Serverless" card
```

Reply:
I didn't understand the suggestion @karthikjeeyar

Reply:
I can update it to
```suggestion
             When user selects on "Broker" from Serverless card
```

Reply:
We can reuse it for channel as well